src/Services/CacheProcessingService.php

```diff
@@ -95,6 +94,13 @@ private function updateEdge(EdgeUpdateDto $dto): array
         }
 
         if ($relation === 'has_one') {
+            $idValue = $instance->getField($edge->getRelation().'ID');
+
+            // A relationship field does exist here, but there is no relationship active
+            if (!$idValue) {
+                return [];
+            }
+
```

Comment:
Was triggering an error on `alreadyProcessed()` as it expects `int` but was being given `null`.